aws/ecs/cloudformation-identiorca.json

```diff
@@ -155,16 +157,12 @@
         "Yes"
       ]
     },
-    "UseScopeService": {
-      "Fn::Not": [
+    "UseWeaveScopeStandalone": {
```

Comment:
Unfortunately it's not that simple, we need to have a conditioned defined, and then we can use `"Condition": "UseWeaveScopeStandalone"` or `"Fn::If": [ "UseWeaveScopeStandalone", <expr>, <else_expr> ]`. In `Fn::If` blocks we can check `WeaveCloudServiceToken`, but it'd make those longer and cause more duplication, and we cannot do the same for  `Condition` parameter. We have `"Condition": "UseWeaveScopeStandalone"` in `Outputs` section, for example.